shim.js

```diff
@@ -0,0 +1,8 @@
+process.browser = true
+
+if (typeof global.Buffer !== 'undefined') {
+  // running on VSCode debugger
+  global.Buffer = undefined
+}
+
```

Comment:
As we are not using `nodify`, does these `Buffer` related code still needed?

Reply:
+1, IIRC the shim.js was generated by the nodify --HACK script right?

Reply:
Good point. It seems that `process.browser = true;` is still needed though. I removed the rest and things still work.. on android at least :+1:

(I get a "`process.split` undefined" if I remove the `process.browser = true;`)